lib/hanami/helpers/html_helper/html_builder.rb

```diff
@@ -61,14 +62,14 @@ class HtmlBuilder # rubocop:disable Metrics/ClassLength
           h6
           head
           header
+          hgroup
           i
           iframe
           ins
           kbd
           label
           legend
           li
-          link
```

Comment:
It was defined twice. 

Please read the entire commit message: https://github.com/hanami/helpers/pull/103/commits/6736ed902f45a9a0df2c84cb706299fe192a135d 😉